### ISSUE TITLE
values are not converted when changing the unit e.g. in an automation

### ISSUE THREAD

**Issue by bltzr** [ossia/score#622]
i.e. I have my parameter \@unit time.Hz
I create an automation by dragging it to a constraint
then I change the unit in the inspector, to e.g. midinote
then the min and max (values and range) stay the same (they should be converted)
also (but I guess that's more complicated) if there are values in the curve, they should also be converted

**Comment by jcelerier** [comment]
> if there are values in the curve, they should also be converted

That would be quite hard for polar coordinates for instance (since it would not be "simple" curves anymore but they could potentially go backwards)

**Comment by bltzr** [comment]
yes, I understand... at least I guess the end and start values (and min/max) should be converted - is that potentially a problem, too ?

**Comment by jcelerier** [comment]
no that's doable

**Comment by bltzr** [comment]
actually this is also happening in libossia implementations (range is not converted for clipping, so I created an issue there)
